api/MediaDevices.json

```diff
@@ -228,27 +228,70 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/getDisplayMedia",
           "description": "<code>getDisplayMedia()</code>",
           "support": {
-            "chrome": {
-              "version_added": "70",
-              "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
-            },
-            "chrome_android": {
-              "version_added": "70",
-              "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
-            },
+            "chrome": [
+              {
+                "version_added": "72"
+              },
+              {
+                "version_added": "70",
+                "version_removed": "72",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Experimental Web Platform features",
+                    "value_to_set": "Enabled"
+                  }
+                ],
+                "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "72"
+              },
+              {
+                "version_added": "70",
+                "version_removed": "72",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Experimental Web Platform features",
+                    "value_to_set": "Enabled"
+                  }
+                ],
+                "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
+              }
+            ],
```

Comment:
As stated in the [Chrome Status entry](https://www.chromestatus.com/features/6744724455030784) this is NOT yet in Android. I've confirmed this with the feature owners.

Reply:
I also suspect this note doesn't apply to Android. Please don't merge until I've confirmed.

Reply:
Good catch. I've updated it. I also added a note about Firefox for android.